### PR TITLE
feat(cloud 2.0): all updates for Cloud 2.0

### PR DESCRIPTION
**Pull request by charislam**
UI flow updates for Cloud 2.0

### CODE REVIEW

File: cloud/service-operations/database-parameters/customize-configuration.md [hunk 1]
```diff
@@ -26,35 +27,44 @@ the compute resources of a running service.
...
-Regardless of whether the service needs to be restarted or not, a confirmation
-dialog is displayed which lists the parameters that are being modified. Click
-`Confirm` to apply the changes (and restart if necessary).
+When you have modified the configuration parameters that you would like to
+change, click `Apply changes`. For some changes, such as
+`timescaledb.max_background_workers`, the Service needs to be restarted. In this
+case, the button reads `Apply changes and restart`.
+
+A confirmation dialog is displayed which indicates whether a restart is
+required. Click `Confirm` to apply the changes (and restart if necessary).
```
Comment (Loquacity) [inline cloud/service-operations/database-parameters/customize-configuration.md:66]:
```suggestion
required. Click `Confirm` to apply the changes, and restart if necessary.
```

File: cloud/service-operations/database-parameters/customize-configuration.md [hunk 2]
```diff
@@ -26,35 +27,44 @@ the compute resources of a running service.
...
-like to change, click `Apply Changes`. For some changes, such as
-`timescaledb.max_background_workers` (pictured below), the Service needs to be
-restarted. In this case, the button reads `Restart and apply changes`.
 
-Regardless of whether the service needs to be restarted or not, a confirmation
-dialog is displayed which lists the parameters that are being modified. Click
-`Confirm` to apply the changes (and restart if necessary).
+When you have modified the configuration parameters that you would like to
+change, click `Apply changes`. For some changes, such as
+`timescaledb.max_background_workers`, the Service needs to be restarted. In this
```
Comment (Loquacity) [inline cloud/service-operations/database-parameters/customize-configuration.md:62]:
```suggestion
`timescaledb.max_background_workers`, the service needs to be restarted. In this
```

**Review by Loquacity — APPROVED** [review]
Just one non-blocking.

**Comment by Loquacity** [comment]
@Tenorotti When are you planning on sending this live?

**Review by Rajakavitha1 — APPROVED** [review]
I have validated this instructions and it works.

**Review by Rajakavitha1 — APPROVED** [review]
validated the instructions.

**Review by Rajakavitha1 — APPROVED** [review]

**Review by Loquacity — APPROVED** [review]

**Pull request merged**